portfolio/src/main/java/com/google/sps/servlets/LoginLogout.java

```diff
@@ -0,0 +1,46 @@
+/*
+  Mario Morales
+  6 March 2020
+*/
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login-logout")
+public class LoginLogout extends HttpServlet {
```

Comment:
Call it LoginLogoutSeverlet and add javadoc

Reply:
Done: 29fdbd4